stdlib/public/CompatibilityOverride/CompatibilityOverride.h

```diff
@@ -222,6 +229,15 @@ namespace swift {
                                     Override,                                  \
                                 fn, &swift_##name##Impl);                      \
     }                                                                          \
+    /* !!! DO NOT REMOVE !!! */                                                \
+    /* This is a workaround for a bug in the LLVM shrink wrap pass, */         \
```

Comment:
Thanks, I hate it.

Reply:
Is there a radar blocked on the LLVM one to remove the workaround once the bug is fixed?

Reply:
We actually have to land the LLVM fix anyway, so I'll remove the workaround before merging. Here's the PR https://github.com/swiftlang/llvm-project/pull/10122